### ISSUE TITLE
Expansion of model summary when using pretrained weights

### ISSUE THREAD

**Issue by Donyme** [keras-team/keras#14412]
While using pretrained weights, present model.summary(), shows the pretrained model as a name instead of the entire expansion. For example in the following model , efficientnetb3 is the pretrained model . A flag can be added to the same indicating whether there is a need for model expansion or keep it as is.

Model: "model"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
input_2 (InputLayer)         [(None, 512, 512, 3)]     0         
_________________________________________________________________
sequential (Sequential)      (None, 512, 512, 3)       0         
_________________________________________________________________
efficientnetb3 (Functional)  (None, 16, 16, 1536)      10783535  
_________________________________________________________________
global_average_pooling2d (Gl (None, 1536)              0         
_________________________________________________________________
dropout (Dropout)            (None, 1536)              0         
_________________________________________________________________
dense (Dense)                (None, 5)                 7685      
=================================================================
Total params: 10,791,220
Trainable params: 10,703,917
Non-trainable params: 87,303

As of now we are taking the model instance separately to get its summary.

efficientnet = EfficientNetB3(weights="imagenet", 
                              include_top=False, 
                              input_shape=input_shape, 
                              drop_connect_rate=dropout_rate)

efficientnet .summary()

**Comment by sushreebarsa** [comment]
@Donyme In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by sushreebarsa** [comment]
Closing this issue due to lack of recent activity & assuming this issue as resolved .Please feel free to reopen if you still have concern.Thanks!
